ocaml/src/alba.ml

```diff
@@ -435,23 +435,31 @@ let alba_show_object
        >>= fun (hm,r) ->
        match r with
          | None ->
-            Lwt_io.eprintlf "not found"
+            Lwt.fail_with "not found"
```

Comment:
but, it does take the --to-json into account, throwing results in something like
```
{"success":false,
  "error":{ "message":"(Failure \"not found\")", 
            "exception_type":"unknown",
            "exception_code":0
          }
}
```